code/game/objects/structures/window_frame.dm

```diff
@@ -56,6 +56,10 @@
 /obj/structure/window_frame/Destroy()
 	density = FALSE
 	update_nearby_icons()
+	for(var/obj/effect/alien/weeds/weedwall/WW in loc)
+		qdel(WW)
+		new /obj/effect/alien/weeds(loc)
```

Comment:
Why is this a loop if a locate suffices

Reply:
It could be a `locate() in loc` I suppose, but I just copy pasta from like 15 lines up:
```
		for(var/obj/effect/alien/weeds/weedwall/window/W in loc)
			weed_found = TRUE
			break
```
🤷